src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/Checkpoints/PendingConfirmationCollection.cs

```diff
@@ -9,54 +9,59 @@ namespace MassTransit.KafkaIntegration.Checkpoints
     public class PendingConfirmationCollection :
```

Comment:
At what point do we end up with `PendingConfigurationCollection<T>` ?

Reply:
it is ready different now, eg: EventHub using PendingConfirmation (eventArgs) to Checkpoint, when Kafka just provides with Offset, I do not think there is a point to abstract such complexity, as of right now :)